src/popup/router/components/Modals/AccountDetails.vue

```diff
@@ -115,6 +143,8 @@ export default {
     return {
       activeTab: 'tokens',
       tabs: ['tokens', 'transactions', 'names'],
```

Comment:
I suggest to not use magic strings. Instead maybe some constants would be better.